docs/topics/proc_deploying-the-caching-booster-using-the-oc-cli-client.adoc

```diff
@@ -0,0 +1,112 @@
+[#deploying-the-crud-booster-using-the-oc-cli-client_{context}]
+= Deploying the {name-mission-crud} Booster using the `oc` CLI Client
+
+.Prerequisites
+
+* The booster application created using {parameter-deployment}.
+ifndef::parameter-openshiftlocal[For more information, see xref:deploying-the-booster-using-launcher-oso_{context}[].]
+ifdef::parameter-openshiftlocal[]
+For more information, see xref:deploying-the-booster-using-launcher-osl_{context}[].
+* Your {launcher} tool URL.
+endif::[]
+
+* The `oc` client authenticated. For more information, see xref:authenticating-the-oc-cli-client_{context}[].
+
+.Procedure
+
+. Create a new project.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc new-project {project-name}
+----
+
+. Navigate to the root directory of your booster.
+
+ifdef::caching-wf-swarm,caching-nodejs[]
+. Deploy the cache service
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc create -f greeting-service/.openshiftio/service.jdg.yaml
```

Comment:
It's `service.cache.yaml` now, but I think we should add the same file to the root directory of the booster.